sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/query.sql

```diff
@@ -0,0 +1,52 @@
+-- Aggregated users data joined with Country Lookup
+SELECT
+  submission_date,
+  country.name AS country,
+  city,
+  activity_segment AS segment,
+  normalized_app_name AS app_name,
+  app_version AS app_version,
+  normalized_channel AS channel,
+  distribution_id AS distribution,
+  normalized_os AS os,
+  normalized_os_version AS os_version,
+  os_version_major,
+  os_version_minor,
+  locale AS language,
+  attribution_content,
+  attribution_source,
+  attribution_medium,
+  attribution_campaign,
+  attribution_experiment,
+  attribution_variation,
+  COUNT(DISTINCT client_id) AS mau,
+  APPROX_COUNT_DISTINCT(IF(days_since_seen < 7, client_id, NULL)) AS wau,
+  APPROX_COUNT_DISTINCT(IF(days_since_seen = 0, client_id, NULL)) AS dau
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.unified_metrics_v1` metrics
+LEFT JOIN
+  `moz-fx-data-shared-prod.static.country_codes_v1` country
```

Comment:
@ANich This doesn't look like the country clean up method you setup. Is it?

Reply:
Okay, I did it here already to speed up the access in Looker, but you have a point about the recomputing which would be a big dependency. So we agree to do it in Looker, right?